File: test/fixtures/output/kotlin/okhttp/jsonObj-multiline.kt
val client = OkHttpClient()

val mediaType = MediaType.parse("application/json")
val body = RequestBody.create(mediaType, "{\n  \"foo\": \"bar\"\n}")
val request = Request.Builder()
  .url("https://httpbin.org/anything")
  .post(body)
  .addHeader("content-type", "application/json")
  .build()

val response = client.newCall(request).execute()

File: src/targets/ruby/native/fixtures/jsonObj-multiline.rb
require 'uri'
require 'net/http'

url = URI("https://httpbin.org/anything")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Post.new(url)
request["content-type"] = 'application/json'
request.body = "{\n  \"foo\": \"bar\"\n}"

response = http.request(request)
puts response.read_body